### ISSUE TITLE
max_len keyword argument appears broken?

### ISSUE THREAD

**Issue by radiaph** [ponyorm/pony#300]
```python
class Foo(db.Entity):
  bar = Required(str, max_len=50)
db.generate_mapping(create_tables=True)
```

Result:
```
Traceback (most recent call last):
  File "test.py", line 15, in <module>
    db.generate_mapping(create_tables=True)
  File "<string>", line 2, in generate_mapping
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/utils/utils.py", line 58, in cut_traceback
    return func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/orm/core.py", line 817, in generate_mapping
    columns = attr.get_columns()  # initializes attr.converters
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/orm/core.py", line 2172, in get_columns
    attr.converters = [ provider.get_converter_by_attr(attr) ]
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/orm/dbapiprovider.py", line 260, in get_converter_by_attr
    return converter_cls(provider, py_type, attr)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/orm/dbapiprovider.py", line 399, in __init__
    Converter.__init__(converter, provider, py_type, attr)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/orm/dbapiprovider.py", line 341, in __init__
    for option in kwargs: throw(TypeError, 'Attribute %s has unknown option %r' % (attr, option))
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pony/utils/utils.py", line 97, in throw
    raise exc
TypeError: Attribute Foo.bar has unknown option 'max_len'
```

Passing max_len as a positional argument still works:

`bar = Required(str, 50)`

**Comment by sashaaero** [comment]
Hello! Seems like you are using old version of pony. We've released 0.7.3 version couple of days ago, so you can update pony via ``python -m pip install --upgrade pony``. This bug was fixed in this release.

**Comment by kozlovsky** [comment]
Fixed in cf4aede6
